ui/src/app/machines/views/MachineDetails/MachineStorage/AvailableStorageTable/BulkActions/BulkActions.tsx

```diff
@@ -0,0 +1,59 @@
+import { Button, Tooltip } from "@canonical/react-components";
+
+import type { BulkAction } from "../AvailableStorageTable";
+
+import CreateVolumeGroup from "./CreateVolumeGroup";
+
+import type { Disk, Machine, Partition } from "app/store/machine/types";
+import { canCreateVolumeGroup } from "app/store/machine/utils";
+
+type Props = {
+  bulkAction: BulkAction | null;
+  selected: (Disk | Partition)[];
+  setBulkAction: (bulkAction: BulkAction | null) => void;
+  systemId: Machine["system_id"];
+};
+
+const BulkActions = ({
+  bulkAction,
+  selected,
+  setBulkAction,
+  systemId,
+}: Props): JSX.Element | null => {
+  const createVgEnabled = canCreateVolumeGroup(selected);
+
+  if (bulkAction === "createVolumeGroup") {
+    return (
+      <CreateVolumeGroup
+        closeForm={() => setBulkAction(null)}
+        selected={selected}
+        systemId={systemId}
+      />
+    );
+  }
+  return (
+    <ul className="p-inline-list u-no-margin--bottom">
+      <li className="p-inline-list__item">
+        <Tooltip
+          data-test="create-volume-group"
+          message={
+            !createVgEnabled
+              ? "Select one or more unpartitioned and unformatted storage devices to create a volume group."
+              : null
+          }
+          position="top-left"
+        >
+          <Button
+            appearance="neutral"
```

Comment:
Should this be positive?

Reply:
I don't think so, it'll be alongside a few other buttons with equal importance and I don't think they'd all be positive. Besides I'm just reimplementing how it is in the angular version.